src/components/SearchModal/ImportToken.tsx

```diff
@@ -47,22 +50,25 @@ export function ImportToken(props: ImportProps) {
       <PaddedColumn gap="14px" style={{ width: '100%', flex: '1 1' }}>
         <RowBetween>
           {onBack ? <ArrowLeft style={{ cursor: 'pointer' }} onClick={onBack} /> : <div />}
-          <ThemedText.MediumHeader>
-            <Plural value={tokens.length} one="Import token" other="Import tokens" />
-          </ThemedText.MediumHeader>
+          <ThemedText.MediumHeader>Import token</ThemedText.MediumHeader>
```

Comment:
I'm cheating here - let me know if this is bad. But I've never seen this modal received more than one token. When would that happen

Reply:
Not sure when/if that would happen, but this will still need to be wrapped in `Trans` or `Plural` for translations.